vNext/UnityGLTFv2.cs

```diff
@@ -0,0 +1,181 @@
+/// <summary>
+/// Handles importing object from schema into Unity and handles exporting of objects from Unity into schema
+/// </summary>
+public partial class UnityGLTFImporter
+{
+    public UnityGLTFImporter(
+        IDataLoader dataLoader,

```

Comment:
I like it being an interface because then we can easily provide default implementation. I suppose we could do the same with default classes but this seems cleaner.

Reply:
I also envision that someone will make a class like 

`public class WebRequestResolver : IDataLoader, IDataWriter`

Reply:
Default implementations of a Func could easily be provided as static methods of a static class. However, if you expect it to be common to share logic across the IDataLoader and IDataWriter, or if you expect the implementation to be stateful, then I feel convinced that an interface is reasonable.